PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -73,6 +73,10 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
     bidRequest.extPrebid.storedAuctionResponse  = Prebid.shared.storedAuctionResponse;
     bidRequest.extPrebid.dataBidders            = self.targeting.accessControlList;
     bidRequest.extPrebid.storedBidResponses     = [Prebid.shared getStoredBidResponses];
+
+    if (Prebid.shared.pbsDebug) {
```

Comment:
Hi @AnthonyTTaylor! Thanks for the fix! 

Please add a test that verifies setting  `Prebid.shared.pbsDebug = true` leads to the `"test": 1` in the request. As an example you can take a look at `ParameterBuilderServiceTest.testBuildParamsDict`

Reply:
Hey @YuriyVelichkoPI 
Added the test, let me know if you need anything else.